.github/workflows/ci.yml

```diff
@@ -0,0 +1,53 @@
+name: Lint and Test Charts
+on:
+  push:
+    branches:
+      - 'main'
+  pull_request: {}
+
+# Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+jobs:
+  setup-terraform:
+    runs-on: self-hosted
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      # creating required environment variables for openstack connection
+      - name: Setup dependencies
+        run: |
+          SUSEConnect -p sle-module-basesystem/15.3/x86_64
+          zypper install -y unzip-6.00-4.8.13
+          # developer-s390-openrc.sh
+          echo $OS_IDENTITY_API_VERSION
+          echo $OS_NETWORK_API_VERSION
+
+      # Install terraform on self-hosted VM if it doesn't exist
+      - name: Install Terraform
+        uses: hashicorp/setup-terraform@v1
+        with:
+          terraform_version: 1.1.4
+
+  deploy-s390x-vm:
+    needs: setup-terraform
+    runs-on: self-hosted
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      # Run terraform script to launch s390x-VM
+      - name: Execute Terraform
+        env:
+          stack_name: "s390x-charts-validation"
```

Comment:
I think the `env` varible names must be prefixed with `TF_VAR_` like `TF_VAR_stack_name` and `TF_VAR_workers`. Also their definitions should be removed from `openstack_tf_s390x/terraform.tfvars` file (would be overritten then).

Reply:
ok, thanks @thehejik